src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -112,10 +112,12 @@
 					/>
 				</div>
 				<kv-pagination
+					ref="categoryPagination"
 					v-if="totalCount > 0"
 					:total="totalCount"
 					:limit="limit"
 					:offset="offset"
+					:scroll-to-top="true"
```

Comment:
This scroll to top doesn't seem to be working when I test locally.

Reply:
It works to me when clicking a page number or prev/next buttons

Reply:
Oh I was testing the lending-home, oops. Actually though I don't think we need this prop set, since the new method calls the page change event, which changes the URL, causing the page to jump to the top. So I think we can remove this line.

Reply:
makes sense, removed!